readthedocs/doc_builder/backends/mkdocs.py

```diff
@@ -59,7 +59,7 @@ def append_conf(self, **kwargs):
         # Mkdocs needs a full domain here because it tries to link to local media files
         if not media_url.startswith('http'):
             domain = getattr(settings, 'PRODUCTION_DOMAIN')
-            media_url = 'http://{}/{}'.format(domain, media_url)
+            media_url = 'http://{}{}'.format(domain, media_url) # media_url has already a prefix slash
```

Comment:
Could you also fix these warnings?

```
doc_builder/backends/mkdocs.py
  Line: 62
    pep8: E261 / at least two spaces before inline comment (col 64)
    pep8: E501 / line too long (102 > 100 characters) (col 101)
```

See https://travis-ci.org/rtfd/readthedocs.org/jobs/152865582#L190 for details.